core/src/main/scala/replpp/util/package.scala

```diff
@@ -42,4 +42,18 @@ package object util {
     }
   }
 
+  /** Lookup the current terminal width - useful e.g. if you want to render something using the maximum space available.
+    * Uses jline-jna, which therefor needs to be in the repl's classpath, which is why it's listed in
+    * {{{replpp.Config.ForClasspath.DefaultInheritClasspathIncludes}}} */
+  def terminalWidth: Try[Int] = {
```

Comment:
we could use `Using(org.jline.terminal.TerminalBuilder.terminal)(_.getWidth)` here, to avoid the platform-dependent switch.

Not sure how heavy that terminal-allocation/close is though. I also haven't checked if that's a different dependency?

Reply:
Thank you, yes, much easier!